File: src/electronicsShop.scala
    def getMoneySpent(keyboards: Array[Int], drives: Array[Int], b: Int): Int = {
        /*
         * Write your code here.
         */
         val cross = keyboards.flatMap(x => drives.map(y => (x, y)))
         val lessThan = cross.filter(_ <= b)
         if lessThan.size == 0 return -1
         return lessThan.max
    }

File: src/electronicsShop.py
#!/bin/python3

import os
import sys

#
# Complete the getMoneySpent function below.
#
def getMoneySpent(keyboards, drives, b):
    most = 0
    for k in keyboards:
        for d in drives:
            print(k+d)
            if k+d <= b and k+d > most:
                most = k+d
    if most != 0:
        return most
    return -1


if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    bnm = input().split()

    b = int(bnm[0])

    n = int(bnm[1])

    m = int(bnm[2])

    keyboards = list(map(int, input().rstrip().split()))

    drives = list(map(int, input().rstrip().split()))

    #
    # The maximum amount of money she can spend on a keyboard and USB drive, or -1 if she can't purchase both items
    #

    moneySpent = getMoneySpent(keyboards, drives, b)

    fptr.write(str(moneySpent) + '\n')

    fptr.close()

